File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SchsSCH0201U99LayoutCommonListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
	public class SchsSCH0201U99LayoutCommonListInfo
	{
		private String _doctorName;
		private String _reserYn;

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String ReserYn
		{
			get { return this._reserYn; }
			set { this._reserYn = value; }
		}

		public SchsSCH0201U99LayoutCommonListInfo() { }

		public SchsSCH0201U99LayoutCommonListInfo(String doctorName, String reserYn)
		{
			this._doctorName = doctorName;
			this._reserYn = reserYn;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SchsSCH0201U99LayoutCommonListInfo.java
package nta.med.data.model.ihis.schs;

public class SchsSCH0201U99LayoutCommonListInfo {
	private String doctorName;
	private String reserYn;

	public SchsSCH0201U99LayoutCommonListInfo(String doctorName, String reserYn) {
		super();
		this.doctorName = doctorName;
		this.reserYn = reserYn;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public String getReserYn() {
		return reserYn;
	}

	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}

}
